java/sort/quicksort/Quicksort.java

```diff
@@ -0,0 +1,65 @@
+package quicksort;
+
+/*
+ * Find the source and further details on the implementation at the below link.
+ * http://www.vogella.com/tutorials/JavaAlgorithmsQuicksort/article.html#quicksort_implementation
+ */
+
+public class Quicksort {
+	private int[] numbers;
+	private int number;
+
+	public void sort(int[] values) {
```

Comment:
Since everything is private, it should return something. 

Also add a method "test" to validate your method. Call that in main method.